lago/providers/libvirt/vm.py

```diff
@@ -341,7 +342,7 @@ def extract_paths(self, paths, ignore_nopath):
                 paths=paths,
                 ignore_nopath=ignore_nopath,
             )
-        except ExtractPathError as err:
+        except (ExtractPathError, SCPException) as err:
```

Comment:
the scp.py module can throw other exceptions(including IOError),
see: https://github.com/jbardin/scp.py/issues/82